packages/web-react/src/components/ScrollView/ScrollViewArrows.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Direction } from '../../constants';
+import { useStyleProps } from '../../hooks';
+import { type SpiritScrollViewArrowsProps } from '../../types';
+import { mergeStyleProps } from '../../utils';
+import { ControlButton } from '../ControlButton';
+import { Icon } from '../Icon';
+import { useScrollViewStyleProps } from './useScrollViewStyleProps';
+
+const ScrollViewArrows = (props: SpiritScrollViewArrowsProps) => {
+  const { direction, scrollStep, viewportRef, ...restProps } = props;
+  const isHorizontal = direction === Direction.HORIZONTAL;
+  const { classProps } = useScrollViewStyleProps({
+    direction,
+  });
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+  const mergedStyleProps = mergeStyleProps('button', { classProps: classProps.arrows, styleProps });
+
+  const handleScroll = (step: number) => {
+    viewportRef.current?.scrollBy({
+      left: isHorizontal ? step : 0,
+      top: isHorizontal ? 0 : step,
+      behavior: 'smooth',
+    });
+  };
```

Comment:
```suggestion
const handleScroll = (step: number) => {
  const axis = isHorizontal ? 'left' : 'top';
  viewportRef.current?.scrollBy({
    [axis]: step,
    behavior: 'smooth',
  });
};
```

Is it possible using a dynamic key (left or top) instead of repeating conditional logic?